var/spack/repos/builtin/packages/py-dask/package.py

```diff
@@ -120,6 +124,7 @@ class PyDask(PythonPackage):
     # Requirements for dask.diagnostics
     depends_on('py-bokeh',              type=('build', 'run'), when='+diagnostics')
     depends_on('py-bokeh@1.0.0:',       type=('build', 'run'), when='@2.0.0: +diagnostics')
+    depends_on('py-bokeh@1.0.0:1.999,2.0.1:', type=('build', 'run'), when='@2020.12.0: +diagnostics')
```

Comment:
If I'm not mistaken, the developers introduced the constraint starting version `2.26.0`:
```python
depends_on('py-bokeh@1.0.0:1.999,2.0.1:', type=('build', 'run'), when='@2.26.0: +diagnostics')
```